ui/packages/ui/src/main/webapp/templates/installer/finish.handlebars

```diff
@@ -31,7 +31,7 @@
                 Click the 'Restart Now' button below to conclude the setup and restart the system.
                 <br/>
                 <br/>
-                Click the 'Shutdown Now' button below to shutdown now. A manual restart will be required to use the system.
+                Click the 'Restart Later' button below avoid restarting. A manual restart will be required to use the system.
```

Comment:
✏️ `below` -> `below to`
❓ Is it really still necessary to restart the system?

Reply:
You won't initiate a restart when you select Restart Later. Not sure how safe that is if the user has made changes during installation, but will be useful for developers I imagine.